src/components/shared/CallNodeContextMenu.css

```diff
@@ -29,8 +29,13 @@
 }
 
 .callNodeContextMenuLabel {
+  display: block;
+  overflow: hidden;
+  width: 250px;
   color: #555;
   font-weight: bold;
+  text-overflow: ellipsis;
+  white-space: nowrap;
 }
```

Comment:
Instead of these changes, I propose this:
```css
.react-contextmenu {
  max-width: 600px;
}
.react-contextmenu-item {
  text-overflow: ellipsis;
  overflow: hidden;
}
```
that you can add in `res/css/style.css` in the respective existing blocks.

Reason is:
* the label as a "block" is not really nice
* the new fix is more generic and would work for other cases that we haven't found yet

What do you think ?

Reply:
I think the above changes is more generic. We could go for that.